src/physics/mp_driver.f90

```diff
@@ -149,7 +149,7 @@ subroutine mp_var_request(options)
         elseif (options%physics%microphysics==kMP_MORRISON) then
             stop "Morrison physics not re-implemented yet"
         elseif (options%physics%microphysics==kMP_WSM6) then
-            stop "WSM6 physics not re-implemented yet"
+            call mp_thompson_aer_var_request(options)
```

Comment:
Can you copy this function into a new mp_wsm6_var_request then remove the variables not required (e.g. number concentrations). That way they won't be allocated, advected, and written to output files unnecessarily).